src/server/analytic_services/bucket_stats_store.js

```diff
@@ -71,6 +71,9 @@ class BucketStatsStore {
         return [];
     }
 
+    async delete_stats({ system, bucket }) {
+        await this._bucket_stats.deleteOne({ system: system, bucket: bucket });
```

Comment:
You don't really need this. try this:
```suggestion
        await this._bucket_stats.deleteOne({ system, bucket });
```

Reply:
Done